src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,364 @@
+  list-style-type: none;
+
+  // added so that the list will be read by screen reader
+  &::before {
+    width: 1px;
+    content: "";
+  }
+}
+
+// Items inside of list
+.pf-c-sequence__item {
+  max-width: var(--pf-c-sequence__item--MaxWidth);
+  margin-right: var(--pf-c-sequence__item--MarginRight);
+  margin-left: var(--pf-c-sequence__item--MarginLeft);
+}
+
+// Vertically centers number and border
+.pf-c-sequence__number-container {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+// Sequence Link Position Number
+.pf-c-sequence__number {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-width: var(--pf-c-sequence__number--MinWidth);
+  max-height: var(--pf-c-sequence__number--MaxHeight);
+  padding: var(--pf-c-sequence__number--Padding);
+  margin-right: var(--pf-c-sequence__number--MarginRight);
+  font-size: var(--pf-c-sequence__number--FontSize);
+  color: var(--pf-c-sequence__number--Color);
+  text-align: center;
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border: var(--pf-c-sequence__number--BorderWeight) solid var(--pf-c-sequence__number--BorderColor);
+    border-radius: var(--pf-c-sequence__number--BorderRadius);
+  }
+}
+
+// Content
+.pf-c-sequence__content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: var(--pf-c-sequence__content--MinHeight);
+}
+
+// Content Title
+.pf-c-sequence__content-title {
+  display: block;
+  max-width: var(--pf-c-sequence__content--title--MaxWidth);
+  overflow: hidden;
+  font-weight: var(--pf-c-sequence__content-title--FontWeight);
+  color: var(--pf-c-sequence__content-title--Color);
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+
+// Content Status
+.pf-c-sequence__content-status {
+  font-size: var(--pf-c-sequence__content-status--FontSize);
+  color: var(--pf-c-sequence__content-status--Color);
+}
+
+// Link
+.pf-c-sequence__link {
+  display: flex;
+  justify-content: flex-start;
+  padding:
+    var(--pf-c-sequence__link--PaddingTop)
+    var(--pf-c-sequence__link--PaddingRight)
+    var(--pf-c-sequence__link--PaddingBottom)
+    var(--pf-c-sequence__link--PaddingLeft);
+  border-bottom: var(--pf-c-sequence__link--BorderBottomHeight) solid var(--pf-c-sequence__link--BorderBottomColor);
+
+  &.pf-m-hover,
+  &:hover {
+    border-bottom: var(--pf-c-sequence__link--BorderBottomHeight) solid var(--pf-c-sequence__link--hover--BorderBottomColor);
+
+    .pf-c-sequence__content-title {
+      color: var(--pf-c-sequence__content-title--hover--Color);
+    }
+
+    .pf-c-sequence__number {
+      // min-width: var(--pf-c-sequence__number--hover--MinWidth);
+      // max-height: var(--pf-c-sequence__number--hover--MaxHeight);
+      // margin-right: var(--pf-c-sequence__number--hover--MarginRight);
+      // font-size: var(--pf-c-sequence__number--hover--FontSize);
```

Comment:
can you remove the commented out lines, and the rest below?

Reply:
yes